src/storage/src/hummock/state_store.rs

```diff
@@ -81,6 +81,10 @@ impl HummockStorage {
         T: HummockIteratorType,
     {
         let epoch = read_options.epoch;
+        let compaction_group_id = match read_options.table_id {
+            None => None,
+            Some(ref table_id) => Some(self.get_compaction_group_id(*table_id).await?),
```

Comment:
why we need a client to get group id? I think this mapping shall be store in `LocalVersionManager` and be updated with version.

Reply:
If mapping for a key is not found in client' cache, the whole mapping will be synced.